services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerConfig.java

```diff
@@ -553,6 +561,11 @@ public VeniceControllerConfig(VeniceProperties props) {
         props.getBoolean(KME_REGISTRATION_FROM_MESSAGE_HEADER_ENABLED, false);
     this.enableDisabledReplicaEnabled = props.getBoolean(CONTROLLER_ENABLE_DISABLED_REPLICA_ENABLED, false);
 
+    this.unusedValueSchemaCleanupServiceEnabled =
+        props.getBoolean(CONTROLLER_UNUSED_VALUE_SCHEMA_CLEANUP_ENABLED, false);
+    this.unusedSchemaCleanupIntervalMinutes = props.getInt(CONTROLLER_UNUSED_SCHEMA_CLEANUP_INTERVAL_MINS, 600);
```

Comment:
So the default interval is 600mins = 10 hours?

Reply:
yes, is that too large ?

Reply:
That should be fine considering schema cleanup is not an urgent task.